components/ui/chart.tsx

```
@@ -1,365 +1,405 @@
-            ([theme, prefix]) => `
+    const colorConfig = Object.entries(config).filter(
+        ([_, config]) => config.theme || config.color
+    );
+
+    if (!colorConfig.length) {
+        return null;
+    }
+
+    return (
+        <style
+            dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)